apps/tx-builder/src/hooks/useServices/interfaceRepository.ts

```diff
@@ -72,6 +72,10 @@ class InterfaceRepository {
           return false;
         }
 
+        if(e?.type.toLowerCase() !== 'function') {
```

Comment:
We should also support `fallback` and `receive` functions (see https://docs.soliditylang.org/en/v0.7.5/abi-spec.html#json)

Reply:
This should also impact the eth field

Reply:
Thanks for the feedback! I improved it now